htlcswitch/link.go

```diff
@@ -1069,9 +1097,31 @@ func (l *channelLink) handleDownStreamPkt(pkt *htlcPacket, isReProcess bool) {
 					"reprocessing queue, batch: %v",
 					htlc.PaymentHash[:],
 					l.batchCounter)
+				debug_print(fmt.Sprintf("Downstream htlc add update with "+
+					"payment hash(%x) have been added to "+
+					"reprocessing queue, batch: %v",
+					htlc.PaymentHash[:],
+					l.batchCounter))
 
 				l.overflowQueue.AddPkt(pkt)
 				return
+			case lnwallet.ErrBelowChanReserve:
+				// CHECK: if the flag is off, then will just fall through to the default case.
+				if (SPIDER_FLAG) {
+					l.infof("Downstream htlc add update with "+
```

Comment:
So the logs are supposed to go in the /logs/ directory - but I couldn't get these to work with the testbench I was using (maybe because it uses a MockServer...). So instead I just added the debug_print method which just prints to stdout if the DEBUG flag is on in htlcswitch/spider_utils.go